### PR TITLE
first PR on feature branch

### PR DESCRIPTION
**Pull request by rosalsm**

### CODE REVIEW

**Comment by ericksonk** [comment]
Looks good :+1: :octocat:

**Comment by boshnivolo** [comment]
Nice! :+1:

File: src/index.html [hunk 1]
```diff
@@ -15,12 +15,15 @@
         <script src="js/vendor/modernizr-2.8.3.min.js"></script>
     </head>
     <body>
-        <!--[if lt IE 8]>
-            <p class="browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
-        <![endif]-->
+      <section class="container">
```
Comment (kellymurray) [inline src/index.html:18]: What other semantic HTML can you use instead of `<sections>` within `<sections>`?

File: src/scss/main.scss [hunk 2]
```diff
@@ -10,91 +10,11 @@
    Base styles: opinionated defaults
```
Comment (kellymurray) [inline src/scss/main.scss:10]: Hmm you don't need this file here...

**Pull request merged**